picard/util/tags.py

```diff
@@ -57,6 +57,12 @@
 DocumentLink = namedtuple('DocumentLink', ('title', 'link'))
 
 
+def _markdown(text: str):
+    if markdown is None:
+        return '<p>' + text.replace('\n', '<br />') + '</p>'
```

Comment:
We should have HTML escaping here to secure against accidental HTML reserved characters in `text`.